src/app/reducers/templates/redux/template.reducers.ts

```diff
@@ -352,9 +352,19 @@ export const selectFilteredTemplates = createSelector(
     const queryLower = listFilters.query && listFilters.query.toLowerCase();
 
     const selectedTypesFilter = ((template: BaseTemplateModel) => {
+      const selfFilter = !!typesMap[TemplateFilters.self]
+        && (template.account === user.name && template.domainId === user.domainid);
+      const sharedFilter = (typesMap[TemplateFilters.sharedExecutable]
```

Comment:
This is the wrong behavior, you get all ready templates from other domains.
![screenshot from 2018-07-27 09-15-18](https://user-images.githubusercontent.com/18456914/43298389-00a795c6-9180-11e8-98eb-55daa9b1e688.png)
![screenshot from 2018-07-27 09-15-26](https://user-images.githubusercontent.com/18456914/43298393-0340bc5e-9180-11e8-9e3c-b4ad2f53a157.png)